packages/qcheck-alcotest/qcheck-alcotest.0.19/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis: "Alcotest backend for qcheck"
+maintainer: "simon.cruanes.2007@m4x.org"
+authors: "the qcheck contributors"
+license: "BSD-2-Clause"
+tags: ["test" "quickcheck" "qcheck" "alcotest"]
+homepage: "https://github.com/c-cube/qcheck/"
+doc: "http://c-cube.github.io/qcheck/"
+bug-reports: "https://github.com/c-cube/qcheck/issues"
+depends: [
+  "dune" {>= "2.2"}
+  "base-bytes"
+  "base-unix"
+  "qcheck-core" {= version}
+  "alcotest"
```

Comment:
```suggestion
  "alcotest" {>= "0.8.1"}
```
```
#=== ERROR while compiling qcheck-alcotest.0.19 ===============================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.09.1 | pinned(https://github.com/c-cube/qcheck/archive/v0.19.tar.gz)
# path                 ~/.opam/4.09/.opam-switch/build/qcheck-alcotest.0.19
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p qcheck-alcotest -j 31
# exit-code            1
# env-file             ~/.opam/log/qcheck-alcotest-4264-64bd85.env
# output-file          ~/.opam/log/qcheck-alcotest-4264-64bd85.out
### output ###
# File "/home/opam/.opam/4.09/lib/alcotest/alcotest.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
#       ocamlc src/alcotest/.qcheck_alcotest.objs/byte/QCheck_alcotest.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.09/bin/ocamlc.opt -w -40 -w +a-4-42-44-48-50-58-32-60@8 -safe-string -g -bin-annot -I src/alcotest/.qcheck_alcotest.objs/byte -I /home/opam/.opam/4.09/lib/alcotest -I /home/opam/.opam/4.09/lib/astring -I /home/opam/.opam/4.09/lib/bytes -I /home/opam/.opam/4.09/lib/cmdliner -I /home/opam/.opam/4.09/lib/fmt -I /home/opam/.opam/4.09/lib/qcheck-core -I /home/opam/.opam/4.09/lib/qcheck-core/runner -I /home/opam/.opam/4.09/lib/result -intf-suffix .ml -no-alias-deps -o src/alcotest/.qcheck_alcotest.objs/byte/QCheck_alcotest.cmo -c -impl src/alcotest/QCheck_alcotest.ml)
# File "src/alcotest/QCheck_alcotest.ml", line 63, characters 8-22:
# 63 |         Alcotest.failf "negative test '%s' succeeded unexpectedly" name
#              ^^^^^^^^^^^^^^
# Error: Unbound value Alcotest.failf
# Hint: Did you mean fail?
#     ocamlopt src/alcotest/.qcheck_alcotest.objs/native/QCheck_alcotest.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.09/bin/ocamlopt.opt -w -40 -w +a-4-42-44-48-50-58-32-60@8 -safe-string -g -I src/alcotest/.qcheck_alcotest.objs/byte -I src/alcotest/.qcheck_alcotest.objs/native -I /home/opam/.opam/4.09/lib/alcotest -I /home/opam/.opam/4.09/lib/astring -I /home/opam/.opam/4.09/lib/bytes -I /home/opam/.opam/4.09/lib/cmdliner -I /home/opam/.opam/4.09/lib/fmt -I /home/opam/.opam/4.09/lib/qcheck-core -I /home/opam/.opam/4.09/lib/qcheck-core/runner -I /home/opam/.opam/4.09/lib/result -intf-suffix .ml -no-alias-deps -o src/alcotest/.qcheck_alcotest.objs/native/QCheck_alcotest.cmx -c -impl src/alcotest/QCheck_alcotest.ml)
# File "src/alcotest/QCheck_alcotest.ml", line 63, characters 8-22:
# 63 |         Alcotest.failf "negative test '%s' succeeded unexpectedly" name
#              ^^^^^^^^^^^^^^
# Error: Unbound value Alcotest.failf
# Hint: Did you mean fail?
```

Reply:
Thanks! Yes, I was just adding that - and hitting the next error while testing locally... :sweat_smile: